### PR TITLE
style: update root help menu for more descriptive commands

### PR DESCRIPTION
**Pull request by efekarakus**
Screen Shot 2020-05-18 at 5 29 13 PM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: internal/pkg/cli/group/group.go [hunk 1]
```diff
@@ -3,12 +3,14 @@
...
 package group
 
+// Categories for each top level command in the CLI.
 const (
-	GettingStarted = "Getting Started ✨"
-	Develop        = "Develop 🔧"
+	GettingStarted = "Getting Started 🌱"
+	Develop        = "Develop ✨"
 	Settings       = "Settings ⚙️"
+	Operational    = "Operational 🧐"
```
Comment (kohidave) [inline internal/pkg/cli/group/group.go:14]: What do you think of “operations”??

Also what do these emojis look like on Linux??

Comment (efekarakus) [reply internal/pkg/cli/group/group.go]: > What do you think of “operations”??

Done!

> Also what do these emojis look like on Linux??

Same UI as macOS 😎

**Comment by efekarakus** [comment]
@iamhopaul123 I can change it to `"Open the copilot docs."` so that we don't have to update the text if we ever migrate out of the wiki.

**Comment by kohidave** [comment]
> @iamhopaul123 I can change it to `"Open the copilot docs."` so that we don't have to update the text if we ever migrate out of the wiki.

I like that - I've been meaning to suggest that for a few days.

**Pull request merged**